src/redist/redist.csproj

```diff
@@ -30,6 +30,17 @@
     <ProjectReference Include="..\tool_nuget\tool_nuget.csproj" />
   </ItemGroup>
 
+  <Target Name="PublishAspNetRuntimePackageStore"
+          AfterTargets="CrossgenPublishDir">
+    <ItemGroup>
+      <AspNetRuntimePackageStore Remove="*" />
```

Comment:
i am suspecting that this did not remove the previous contents of AspNetRuntimePackageStore